allauth/account/signals.py

```diff
@@ -9,8 +9,9 @@
 password_changed = Signal(providing_args=["request", "user"])
 password_reset = Signal(providing_args=["request", "user"])
 
-email_confirmed = Signal(providing_args=["email_address"])
-email_confirmation_sent = Signal(providing_args=["confirmation"])
+email_confirmed = Signal(providing_args=["request", "email_address"])
+email_confirmation_sent = Signal(providing_args=["request", "signup",
+                                                 "confirmation"])
```

Comment:
As we are breaking the existing signal signature, this needs a note in the `ChangeLog.rst` (backwards incompatible changes).